modules/test/playwright/helpers/FeatureFlagApiHelper.ts

```diff
@@ -3,12 +3,16 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {Page} from '@playwright/test';
+
 export class FeatureFlagApiHelper {
-	constructor(page) {
+	readonly page: Page;
+
+	constructor(page: Page) {
 		this.page = page;
 	}
 
-	async updateFeatureFlag(key, enabled) {
+	async updateFeatureFlag(key: Object, enabled: boolean) {
```

Comment:
key should be a string, right? changed [here](https://github.com/liferay-objects/liferay-portal/pull/2996/commits/ef7f99628ac8b5b28cca0f4e804789339d4ffc48).

Reply:
By the way, `Object` is a very permissive type. It can be anything but `null` or `undefined`.